research/2022-08-20-introducing-waku-product.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+name:  "Waku for All DApps"
+title:  "Waku for All DApps"
+date:   2022-11-01 00:00:00 +0000
+author: franck
+published: true
+permalink: /waku-for-all-dapps
+categories: waku, dapp, platform, operator
+summary: Waku adoption push has shifted from Web to all platforms. 
```

Comment:
Waku is an open communication protocol that decentralised apps and infrastructure can use for their communication needs. It is designed to enable dapps and decentralised infrastructure projects to access <add characteristics... e.g. scalable, highly performant, plug and play, ....> messaging features. It is accessible through <.... js Waku, nwaku,....> options.